Doc/reference/expressions.rst

```diff
@@ -201,6 +210,12 @@ or :keyword:`await` expressions it is called an
 suspend the execution of the coroutine function in which it appears.
 See also :pep:`530`.
 
+.. versionadded:: 3.6
+   Asynchronous comprehensions were introduced
+
+.. versionchanged:: 3.7
```

Comment:
Shouldn't be `deprecated` or `deprecated-removed`?

Reply:
Aye, it should - I just forgot there was a specific directive for deprecations. I'll go through and fix all the cases.